### ISSUE TITLE
bug/docs: skuid salesforce support in skuid-cli

### ISSUE THREAD

**Issue by techfg** [skuid/skuid-cli#183]
According to the [docs](https://docs.skuid.com/latest/en/skuid/cli/#working-with-skuid-for-salesforce), Skuid SFX and Salesforce-related features have been ***deprecated*** from skuid-cli and users encouraged to migrate to `skuid-sfdx` (see highlights in image below).

However, in reviewing the code, it appears that all `force` related code has been removed entirely (primarily via [this commit](https://github.com/skuid/skuid-cli/commit/ca75f73a8774d13af9322cf4f8885f797876e195)) and is no longer supported (as opposed to `deprecated`).

Does `skuid-cli` still work with Salesforce or are the docs just 16+ months out of date?

![image](https://github.com/user-attachments/assets/f23715d2-ac08-43b5-8615-58965ec4f352)

**Comment by itmightbecody** [comment]
Hey @techfg, you're correct in that using the `skuid` CLI to deploy to Salesforce orgs is unsupported with the latest releases. We could be more accurate by saying that in our docs instead of deprecated. I'll get this updated.

**Comment by techfg** [comment]
Hi @itmightbecody -

Thanks for confirming!  If I'm interpreting the commits properly, SFX support was completely removed in/around May 2023 when it went from `deprecated` to `no longer supported` so would be great to see the docs updated to reflect current state, thank you!

Looking forward to hopefully hearing from you and/or someone on the Skuid team on the other issues I've submitted recently as well :)

Thanks again!

**Comment by itmightbecody** [comment]
Hey @techfg ,
We've made these requested changes to our docs:
- [SFX docs](https://docs.skuid.com/latest/v2/en/skuid/cli/)
- [NLX docs](https://docs.skuid.com/nlx/v2/en/skuid/cli/)

Thanks for your attention to detail here—it's never a bad thing to have more accurate docs.
I've reached out to folks on the product team, and I believe they'll be in contact regarding the other issues you've pinpointed.

All the best,
Cody

**Comment by techfg** [comment]
Hi @itmightbecody -

Thanks for getting the docs updated!  Speaking of attention to detail, not surprisingly given history, the doc updates that were made are not accurate.  Since this issue is closed, I've created #206 to track the issue I found with the update that was made.  I also updated #201 since the changes that were made for this issue made me catch another discrepancy related to that issue.

Regarding the other issues, appreciate you engaging the product team to raise their awareness, looking forward to hearing from them!

Thanks!
